src/components/Icons.js

```diff
@@ -59,43 +57,43 @@ export const BalanceIcon = () => (
                 />
             </g>
         </svg>
-    </Image>
+    </div>
 );
 
-export const StableIcon = () => <Image centered src={stableSVG} />;
```

Comment:
I'd still create an Image component to wrap the `<img />` elements, it would be easier to handle variants in the future.